src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj

```diff
@@ -15,23 +15,30 @@
     <PackageId>Serilog.Settings.Configuration</PackageId>
     <PackageTags>serilog;json</PackageTags>
     <PackageIcon>icon.png</PackageIcon>
-    <PackageProjectUrl>https://github.com/serilog/serilog-settings-configuration</PackageProjectUrl>
+    <PackageProjectUrl>https://serilog.net/</PackageProjectUrl>
```

Comment:
nit : this changes the link that the Nuget page points to  : 

https://www.nuget.org/packages/Serilog.Settings.Configuration/3.1.1-dev-00228

<img width="329" alt="image" src="https://user-images.githubusercontent.com/160544/81534231-fa641900-9367-11ea-80c2-768acf077cdd.png">

Not sure if it's a big deal, but if people is looking for docs specific to that package it becomes a little bit harder to find, what do you thing ?

Reply:
@tsimbalar , ah, right, a bad copy/paste from Serilog repository, will fix in next change

Reply:
I've been going on the assumption that "source repository" is the GitHub link, while "project" is more informational, but either way seems fine to me :-)

Reply:
@nblumhardt you are right. But unfortunately "source repository" is not shown in VS nor in the linqpad 😄 :
![image](https://user-images.githubusercontent.com/15090984/81619595-a18d9280-93f2-11ea-9226-5dc686cdaaaf.png)

![image](https://user-images.githubusercontent.com/15090984/81619620-ac482780-93f2-11ea-9d94-9703a90adb6f.png)